scripts/package/projectsToPack.ps1

```diff
@@ -3,10 +3,13 @@
 
 $ProjectsToPack = @(
     "Microsoft.DotNet.Cli.Utils",
+    "Microsoft.DotNet.Compiler.Common",
+    "Microsoft.DotNet.Files",
```

Comment:
Because of https://github.com/dotnet/cli/issues/1755. ProjectModel.Workspaces depends on Compiler.Common.